src/Workleap.DomainEventPropagation.Subscription.PullDelivery.Tests/EventPullerServiceTests.cs

```diff
@@ -54,93 +57,16 @@ private static void RegisterCloudEventsInClient(IEventGridClientAdapter client,
             });
     }
 
-    private static async Task StartWaitAndStop(IHostedService pullerService)
+    private static async Task StartWaitAndStop(EventPullerService pullerService, int numberOfEventToProcess)
     {
         // We need this to start on the thread pool otherwise it will just block the test
-        Task.Run(() => pullerService.StartAsync(CancellationToken.None)).Forget();
-        await Task.Delay(50);
-        await pullerService.StopAsync(CancellationToken.None);
-    }
-
-    public class TwoSubscribers : EventPullerServiceTests
```

Comment:
Remove these tests, and instead "merging" them with `TestPublishAndReceiveEvent`